examples/non-fungible-token/tests/sim/utils.rs

```diff
@@ -0,0 +1,123 @@
+use approval_receiver::ApprovalReceiverContract;
+use near_contract_standards::non_fungible_token::metadata::TokenMetadata;
+use non_fungible_token::ContractContract as NftContract;
+use token_receiver::TokenReceiverContract;
+
+use near_sdk_sim::{call, deploy, init_simulator, to_yocto, ContractAccount, UserAccount};
+use near_contract_standards::non_fungible_token::token::TokenId;
+
+// Load in contract bytes at runtime
+near_sdk_sim::lazy_static_include::lazy_static_include_bytes! {
+    NFT_WASM_BYTES => "res/non_fungible_token.wasm",
+    TOKEN_RECEIVER_WASM_BYTES => "res/token_receiver.wasm",
+    APPROVAL_RECEIVER_WASM_BYTES => "res/approval_receiver.wasm",
+}
+
+const NFT_ID: &str = "nft";
+const TOKEN_RECEIVER_ID: &str = "token-receiver";
+const APPROVAL_RECEIVER_ID: &str = "approval-receiver";
+
+// TODO: how to export String instead of &str? Way too much `into`/`to_string` with &str.
```

Comment:
This can be done with [lazy_static](https://docs.rs/lazy_static/1.4.0/lazy_static/) or some equivalent. I don't know if it fixes anything, because then it would always have to clone if being moved, but can revisit this later